apps/pie-docs/test/system/navigation.spec.js

```diff
@@ -45,3 +46,33 @@ test.describe('PIE - site nav menu - @mobile', () => {
         await expect(navMenu).not.toBeVisible();
     });
 });
+
+test.describe('PIE - site nav menu - @desktop', () => {
+    let siteNavigation;
+    let contentHeader;
+
+    test.beforeEach(async ({ page, context }) => {
+        await disableCookieBanner(page, context);
+        await page.goto('all-about-pie/what-is-pie/');
+
+        // Arrange
+        siteNavigation = await page.getByTestId('site_nav_section_1');
+        contentHeader = await page.getByTestId('content_header');
+    });
+
+    test('Should be visible at the top of the page', async () => {
+        // Assert - Navigation and top of the page are visible
+        await expect.soft(await isElementVisibleInViewport(siteNavigation)).toBeTruthy();
+        await expect.soft(await isElementVisibleInViewport(contentHeader)).toBeTruthy();
```

Comment:
Each test should end with a normal except. So you can remove the `.soft` from this line so that the test ends.